src/blk/zoned/HMSMRDevice.h

```diff
@@ -41,6 +41,7 @@ class HMSMRDevice final : public BlockDevice {
   string vdo_name;
 
   std::string devname;  ///< kernel dev name (/sys/block/$devname), if any
+  int zbd_dev;
```

Comment:
This isn't initialized atm, suggest to mark with FIXME at least

Reply:
Fixed.